### DISCUSSION TITLE
3.5.0

### DISCUSSION THREAD

**Discussion by nalexandru** [nalexandru/BlenderCompat#36] [Announcements]
Blender 3.5.0, commit [1be25cfff18b](https://projects.blender.org/blender/blender/commit/1be25cfff18b).
Build environment:

    Visual Studio 2019, version 16.11.25
    Windows SDK 10.0.22621
    NVIDIA CUDA 11.8
    NVIDIA OptiX 7.3

AMD HIP and SM3.0 kernel binaries copied from the official release.

<hr /><em>This discussion was created from the release <a href='https://github.com/nalexandru/BlenderCompat/releases/tag/3.5.0'>3.5.0</a>.</em>

**Comment by patientAtGithub** [comment]
Thank you again **VERY** much!

**Comment by lnesuper** [comment]
Please compile blender 3.3.5 LTS version ！！！

**Reply by nalexandru** [reply]
Here you go, #37. I don't use the LTS release so i was not following it. I will build LTS too from now on.
